RecoHI/HiJetAlgos/src/MultipleAlgoIterator.cc

```diff
@@ -160,6 +169,61 @@ void MultipleAlgoIterator::calculatePedestal(vector<fastjet::PseudoJet> const& c
   }
 }
 
+void MultipleAlgoIterator::calculateOrphanInput(vector<fastjet::PseudoJet>& orphanInput) {
+  LogDebug("PileUpSubtractor") << "The subtractor calculating orphan input...\n";
+
+  (*fjInputs_) = fjOriginalInputs_;
+
+  vector<int> jettowers;                   // vector of towers indexed by "user_index"
+  vector<pair<int, int> > excludedTowers;  // vector of excluded ieta, iphi values
+
+  for (auto const& pseudojetTMP : *fjJets_) {
+    if (pseudojetTMP.perp() < puPtMin_)
+      continue;
+
+    // find towers within radiusPU_ of this jet
+    for (auto const im : allgeomid_) {
+      double dr = reco::deltaR(geo_->getPosition(im), pseudojetTMP);
+      vector<pair<int, int> >::const_iterator exclude =
+          find(excludedTowers.begin(), excludedTowers.end(), pair<int, int>(im.ieta(), im.iphi()));
```

Comment:
can be shorter as
```suggestion
      auto exclude = find(excludedTowers.begin(), excludedTowers.end(), {im.ieta(), im.iphi()});
```

Reply:
Doesn't compile with curly brackets: `template argument deduction/substitution failed`